src/Xamarin.Android.Build.Tasks/Utilities/ResourceDesignerImportGenerator.cs

```diff
@@ -59,8 +59,14 @@ public void CreateImportMethods (IEnumerable<ITaskItem> libraries)
 						var declaringTypeName = $"{reader.GetString (declaringType.Namespace)}.{reader.GetString (declaringType.Name)}";
 						if (declaringTypeName == resourceDesignerName) {
 							if (hasAlias) {
-								string [] aliases = alias.Split (new [] {','}, StringSplitOptions.RemoveEmptyEntries);
-								declaringTypeName = $"{aliases[0].Trim ()}::{declaringTypeName}";
+								string [] aliases = aliasMetaData.Split (new [] {','}, StringSplitOptions.RemoveEmptyEntries);
+								foreach (var alias in aliases) {
+									string aliasName = alias.Trim ();
+									if (string.Compare ("global", aliasName, StringComparison.OrdinalIgnoreCase) == 0)
```

Comment:
I'm not certain that `OrdinalIgnoreCase` is appropriate here, as C# is a case sensitive language.  I'm not sure it's "unreasonable" for someone to use `Global` as an alias.

…unless MSBuild metadata is *by design* case insensitive, in which case this is appropriate.  I don't think that's the case (?).